CONTRIBUTING.md

```diff
@@ -159,11 +159,6 @@ If you want to use scopes then it would look more like:
 The commit message formatting described above is automatically enforced
 each time you commit to your work branch to make continuous integration smoother.
 
-**If you're on Windows**, the commit message verification currently doesn't
```

Comment:
These changes were merged to master in https://github.com/mozilla/web-ext/commit/c0dc6744fb4409c7847be2bac2335273ec082ccf \o/ If you pull changes from master and push those changes to your fork of the repository then they won't appear in this pull request.